README.md

```diff
@@ -179,6 +179,9 @@ main()
           - `adapter(partialResult)` (function, optional) - When resolving an entity across multiple subgraphs, an initial query is made to one subgraph followed by one or more followup queries to other subgraphs. The initial query must return enough information to identify the corresponding data in the other subgraphs. This function is invoked with the result of the initial query. It should return an object whose keys correspond to the `primaryKeyFields` configuration.
           - `primaryKeyFields` (array of strings, required) - The fields used to uniquely identify objects of this type.
           - `referenceListResolverName` (string, required) - The name of a resolver used to retrieve a list of objects by their primary keys.
+      - `onSubgraphError` (function, optional) - Hook called when an error occurs getting schema from a subgraph. The default function will throw the error. The arguments are:
+          - `error` (error) - The error.
+          - `subgraph` (error) - The erroring subgraph.
```

Comment:
I don't think this is quite accurate. This says there are two separate arguments, but looking at the following line:

```js
this.onSubgraphError({ subgraph: { ...subgraph }, error: new Error(msg, { cause: responses[i].reason }) })
```

There is a single argument object. Is there any reason not to use two separate arguments?